CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ INCLUDE(GetGitRevisionDescription)
 get_git_head_revision(GIT_REFSPEC FC_GIT_REVISION_SHA)
 get_git_unix_timestamp(FC_GIT_REVISION_UNIX_TIMESTAMP)
 
+SET( CMAKE_CXX_STANDARD 11 )
+
```

Comment:
Maybe we don't need  "-std=c++11" any more? Find "SET(CMAKE_CXX_FLAGS" in projects.